spark-indexer/src/test/java/org/uniprot/store/spark/indexer/validator/impl/SolrIndexValidatorUtil.java

```diff
@@ -0,0 +1,42 @@
+package org.uniprot.store.spark.indexer.validator.impl;
+
+import org.apache.solr.client.solrj.impl.CloudSolrClient;
+import org.apache.solr.client.solrj.response.QueryResponse;
+import org.apache.solr.common.SolrDocumentList;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.mockito.Mockito;
+import org.uniprot.store.spark.indexer.common.JobParameter;
+
+import com.typesafe.config.Config;
+
+public class SolrIndexValidatorUtil {
+
+    static JobParameter getJobParameter(Config applicationConfig, JavaSparkContext context) {
+        return JobParameter.builder()
+                .releaseName("2020_02")
+                .applicationConfig(applicationConfig)
+                .sparkContext(context)
+                .build();
+    }
+
+    static <T extends AbstractSolrIndexValidator> T spyMockValidator(
+            T validatorImpl, long docCount, long solrCount) throws Exception {
```

Comment:
Why are we calling this method spyMockValidator? Basically this method is wrapping mocks.

Reply:
Can you please specify the method name that you prefer. I gave a name that I think it is better, but as you did not like the current name, I prefer you specify the better name for it

Reply:
wrapMockSolrIndexValidator is good